deluge/ui/gtk3/glade/preferences_dialog.ui

```diff
@@ -2573,8 +2573,8 @@ used sparingly.</property>
                                       <object class="GtkEntry" id="entry_interface">
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
-                                        <property name="tooltip_text" translatable="yes">The IP address of the interface to listen for incoming bittorrent connections on. Leave this empty if you want to use the default.</property>
-                                        <property name="max_length">15</property>
+                                        <property name="tooltip_text" translatable="yes">IP addresses / network interface names to be listened for incoming bittorrent connections. Leave this empty if you want to use the default.</property>
```

Comment:
This is using plural but this change is only for singular. Could also be shortened


`IP address or network interface name to listen for incoming BitTorrent connections. Leave empty to use system default.`